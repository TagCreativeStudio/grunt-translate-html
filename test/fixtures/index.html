<!DOCTYPE html>
<html lang="en">
  <head>
    <title>Tag Integrated</title><!--(if target dev)><!-->
    <link href="css/tagintegrated.css" rel="stylesheet" type="text/css" media="screen">
    <script src="js/modernizr.js" type="text/javascript"></script><!--<(endif)-->
    <!--(if target prod)><!-->
    <link href="css/tagintegrated.min.css" rel="stylesheet" type="text/css" media="screen">
    <script src="js/modernizr.min.js" type="text/javascript"></script><!--<!(endif)-->
  </head>
  <body id="body">
    <section class="section-one">
      <h2 data-localize id="super-id" style="text-align:center" class="non-class extra final-class">This is a header</h2>
      <p>This is a paragraph</p>
      <p localize>This is another paragraph</p>
    </section><!--(if target dev)><!-->
    <script src="js/vendor.js" type="text/javascript"></script>
    <script src="js/tagintegrated.js" type="text/javascript"></script><!--<!(endif)-->
    <!--(if target prod)><!-->
    <script src="js/vendor.min.js" type="text/javascript"></script>
    <script src="js/tagintegrated.min.js" type="text/javascript"></script><!--<!(endif)-->
  </body>
</html>