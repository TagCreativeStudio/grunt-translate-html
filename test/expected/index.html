<!DOCTYPE html>
<html lang="de">
  <head>
    <title>Title goes here</title>
  </head>
  <body>
    <section>
      <h2 data-localize>Dies ist eine Kopf</h2>
      <p>This is a paragraph</p>
      <p data-localize>Dies ist ein weiterer Absatz</p>
      <p data-localize>Ich erinnere mich, Sie kennen zu lernen auf düsterer Straße</p>
    </section>
  </body>
</html>